zola.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<component type="console-application">
+  <id>org.getzola.zola</id>
+  <icon type="local">docs/static/favicon.ico</icon>
+
+  <name>Zola</name>
+  <summary>A fast static site generator in a single binary with everything built-in</summary>
+  <developer_name>Vincent Prouillet</developer_name>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <description>
+    <p>
+      Zola comes as a single executable with Sass compilation, syntax highlighting, table of contents and many other features 
+      that traditionally require setting up a dev environment adding some JavaScript libraries to your site.
+    </p>
+    <p>
+      The average site will be generated in less than a second, including Sass compilation and syntax highlighting.
+    </p>
+    <p>
+      Zola renders your whole site as static files, making it trivial to handle any kind of traffic you will throw at it at no 
+      cost without having to worry about managing a server or a database.
+    </p>
+    <p>
+      From the CLI to the template engine, everything is designed to be intuitive. Don't take my word for it though, look at the
+      documentation and see for yourself.
+    </p>
+    <p>
+      Zola gets out of your way so you can focus on your content, be it a blog, a knowledge base, a landing page or a combination 
+      of them.
+    </p>
+    <p>
+      Zola comes with shortcodes and internal links to make it easier to write your content.
+    </p>
+  </description>
+
+  <categories>
+    <category>Development</category>
+    <category>WebDevelopment</category>
+  </categories>
+
+  <url type="homepage">https://www.getzola.org/</url>
+  <url type="bugtracker">https://github.com/getzola/zola/issues</url>
+  <url type="help">https://www.getzola.org/documentation/</url>
+
+  <provides>
+    <binary>zola</binary>
+  </provides>
+
+  <content_rating type="oars-1.1" />
+
+  <releases>
+    <release version="0.15.3" date="2022-01-22">
+      <url>https://github.com/getzola/zola/releases/tag/v0.15.3</url>
+      <description>
+        Zola Static Site Generator v0.15.3
+      </description>
```

Comment:
Since this section doesn't really contain any information about the release, I would suggest you remove it.

Reply:
Sure, you can do that. Here's an example if it's helpful: https://github.com/flathub/com.jpexs.decompiler.flash/blob/master/com.jpexs.decompiler.flash.metainfo.xml#L66

Reply:
@Newbytee I think it would be better to remove the description. Since the changelog is autogenerated, it's not all that useful.

Reply:
Sounds good to me.